src/zkevm_specs/util/param.py

```diff
@@ -1,5 +1,5 @@
 # Maximun number of bytes with composition value that doesn't wrap around the field
-MAX_N_BYTES = 31
+MAX_N_BYTES = 32
```

Comment:
There's no guarantee that 32 bytes don't wrap around the field, so this value should be kept at `31`.

Reply:
last version testing typo, sorry.